Config.cmake/runtime_test.cmake

```diff
@@ -152,9 +152,8 @@ include(CTest)
 set(CTEST_RETURN_SUCCESS 0)
 set(CTEST_RETURN_FAIL 1)
 
-to_native_c_path(${TEST_OUTPUT_FOLDER}/tempfolder FMU_TEMPFOLDER)
-
-file(MAKE_DIRECTORY ${FMU_TEMPFOLDER})
+to_native_c_path(${TEST_OUTPUT_FOLDER}/Temporary TEST_RESULTS_DIR)
+file(MAKE_DIRECTORY ${TEST_RESULTS_DIR})
```

Comment:
.../Temporary seems to be some directory generated by cmake, for cmake files not controlled by, especially test result files. I've changed such that we should never write to that directory.

If we need a temp directory we create it and delete it when done.